### ISSUE TITLE
training data

### ISSUE THREAD

**Issue by zxz-cc** [Megvii-BaseDetection/YOLOX#857]
Is the training data structure must COCO? Can i use VOC structure ?like xml??

**Comment by FateScript** [comment]
Yes, plz refer to [this exp](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/exps/example/yolox_voc/yolox_voc_s.py)
